src/index.html

```diff
@@ -3,6 +3,7 @@
 <link rel="icon" type="image/svg+xml" href="../collection/assets/favicon.svg">
 <head>
     <meta charset="utf-8">
+    <meta name="color-scheme" content="light dark">
```

Comment:
this line is good when we rely on system's color scheme. I think it would have been better to conditionalize this as well, just like how I did in the CSS. So when the `root` has `data-theme='system-default'` we return the tag like above.

But when the `root` has `data-theme='force-light'`, we return `<meta name="color-scheme" content="light">`

and for the dark, we return `content="dark"` only. 

I don't know how to do so in the `index.html` file. If this is not possible, or not so easy, the current one is already alright.

Reply:
I can have a look at adding that 👍

Reply:
nice. wanna do it in this PR or a separate PR?

Reply:
I did it here 😊

Reply:
We came to the conclusion that switching it with javascript is most likely completely redundant, so we removed that commit again.